README.md

```diff
@@ -1,17 +1,15 @@
 # Terragrunt
 
-Terragrunt is a thin wrapper for [Terraform](https://www.terraform.io/) that provides extra tools for working with
-multiple Terraform modules, remote state, and locking.
-
-
+Terragrunt is a thin wrapper for [Terraform](https://www.terraform.io/) that provides extra tools for keeping your Terraform scripts DRY, working with multiple Terraform modules, and managing remote state.
 
 
 ## Quick start
 
-1. Install [Terraform](https://www.terraform.io/).
+1. [Install Terraform](https://www.terraform.io/intro/getting-started/install.html).
 
-1. Install Terragrunt by going to the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases), downloading
-   the binary for your OS, renaming it to `terragrunt`, and adding it to your PATH.
+1. Install Terragrunt
+   * On OSX `brew install terragrunt`. 
+     See the [Install Terragrunt]() docs for other installation options.
```

Comment:
I'm not sure we should recommend homebrew as an official install method because we don't automatically update homebrew packages to the latest Terragrunt version and they're often a little out of date. My preference would be to retain the original wording as the first bullet point and add a second bullet point that indicates that you can also install Terragrunt with various package manages and links to your below details.